doc/manual/rl-next/print-value-in-installable-flake-error.md

```diff
@@ -0,0 +1,18 @@
+---
+synopsis: New-cli flake commands that expect derivations now print the failing value and its type
```

Comment:
```suggestion
synopsis: `nix flake` commands that expect derivations now print the failing value and its type
```

Reply:
Looks like the yaml parser doesn't like backticks. I had to remove them again.